utils/common/src/main/java/org/apache/brooklyn/util/exceptions/Exceptions.java

```diff
@@ -46,12 +47,19 @@
 
 public class Exceptions {
 
-    private static final List<Class<? extends Throwable>> BORING_THROWABLE_SUPERTYPES = ImmutableList.<Class<? extends Throwable>>of(
-        ExecutionException.class, InvocationTargetException.class, PropagatedRuntimeException.class, UndeclaredThrowableException.class);
+    private static final List<Class<? extends Throwable>> ALWAYS_BORING_THROWABLE_SUPERTYPES = ImmutableList.<Class<? extends Throwable>>of(
+        ExecutionException.class, InvocationTargetException.class, UndeclaredThrowableException.class);
```

Comment:
I think it's ok so far with where this is being used, but we need to be very careful to not accidentally imply that there was an exception in the current thread, when in fact it was thrown in a different thread. That kind of thing can be extremely misleading and a huge pain when debugging. We really do want to keep ExecutionException to indicate it was an exception from (probably) a different thread when we're logging.

But when it's to truncate a message for the user, then that's absolutely fine.